roles/lib_utils/library/openshift_container_binary_sync.py

```diff
@@ -31,13 +31,12 @@ class BinarySyncer(object):
     a container onto the host system.
     """
 
-    def __init__(self, module, image, tag, backend):
+    def __init__(self, module, image, backend):
```

Comment:
Need to remove references to the tag parameter in the module spec in this files 'main()' function (line 179 and 186) below.

Reply:
And ~192.